x-pack/packages/kbn-langchain/server/utils/bedrock.ts

```diff
@@ -180,7 +180,7 @@ const prepareBedrockOutput = (responseBody: CompletionChunk, logger?: Logger): s
       return responseBody.delta.text;
     }
   }
-  logger?.warn(`Failed to parse bedrock chunk ${JSON.stringify(responseBody)}`);
```

Comment:
I originally put in the `logger.warn` in there in case there was another type of chunk with text output that I was not accounting for, but this has not proven to be the case, it is way too noisy, and causes false concerns. Thus I am removing it.